tests/Unit/CssInlinerTest.php

```diff
@@ -445,12 +445,31 @@ public function wbrTagDataProvider()
      *
      * @dataProvider wbrTagDataProvider
      */
-    public function inlineCssByDefaultRemovesWbrTag($html)
+    public function inlineCssByDefaultNotRemovesWbrTag($html)
     {
         $subject = $this->buildDebugSubject($html);
 
         $subject->inlineCss('');
 
+        $expectedWbrTagCount = \substr_count($html, '<wbr');
+        $resultWbrTagCount = \substr_count($subject->render(), '<wbr');
+        self::assertSame($expectedWbrTagCount, $resultWbrTagCount);
```

Comment:
Can we move the "execution" phase to a separate line (also in the next test)?

```php
$result = $subject->render();
```

Reply:
Done.